docs/README.md

```diff
@@ -43,13 +43,10 @@
 - throttle vs debounce
 - object creation patterns tutorial - factory , constructor pattern, prototype pattern
 - JavaScript ES2020
+- Local storage vs Session storage
 - Tricky JavaScript interview questions and answers
 
 
-### Good references
-
-1. https://medium.com/javascript-scene/10-interview-questions-every-javascript-developer-should-know-6fa6bdf5ad95
```

Comment:
I think you accidentally deleted this line (pls include it)

Reply:
okay will fix it